examples/hotrod/pkg/pool/pool_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pool
+
+import (
+	"testing"
+	"time"
+)
+
+var n int = 0
+
+func TestExecute(t *testing.T) {
+	testPool := New(1)
+	tests := []struct {
```

Comment:
there is no value is making the test data-driven if there is only one test case involved

Reply:
thank's for your reply, i have update the code.